target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -101,6 +102,19 @@ function __rspamd__run_early_setup_and_checks() {
   fi
 }
 
+function __rspamd__setup_logfile() {
+  cat >/etc/logrotate.d/rspamd << EOF
+/var/log/mail/rspamd.log
+{
+  compress
+  copytruncate
+  delaycompress
+  rotate 4
+  ${LOGROTATE_INTERVAL}
+}
+EOF
+}
```

Comment:
**Optional:** While unlikely to introduce configuration drift, might be wise to add this comment for future maintenance?

```suggestion
# Keep in sync with `target/scripts/startup/setup.d/log.sh:_setup_logrotate()`
function __rspamd__setup_logfile() {
  cat >/etc/logrotate.d/rspamd << EOF
/var/log/mail/rspamd.log
{
  compress
  copytruncate
  delaycompress
  rotate 4
  ${LOGROTATE_INTERVAL}
}
EOF
}
```

Reply:
At a glance beyond the `export` / `VARS` additions to `_setup_logrotate()` I don't see the dependency requirement?

If the ENV was misconfigured we'd bail with a panic now which is more correct. So there is no concern remaining about handling a misconfigured interval ENV reaching this point.